crates/astria-cli/src/commands/sequencer.rs

```diff
@@ -0,0 +1,146 @@
+use astria_sequencer_client::{
+    HttpClient,
+    SequencerClientExt,
+};
+use color_eyre::{
+    eyre,
+    eyre::Context,
+};
+use ed25519_consensus::SigningKey;
+use rand::rngs::OsRng;
+use sha2::{
+    Digest,
+    Sha256,
+};
+
+use crate::cli::sequencer::{
+    BalanceGetArgs,
+    BlockHeightGetArgs,
+};
+
+/// Generate new keypair
+fn get_new_signing_key() -> SigningKey {
+    let csprng = OsRng;
+    let signing_key: SigningKey = SigningKey::new(csprng);
+    signing_key
+}
+
+/// Get the public key from the signing key
+fn get_public_key_pretty(signing_key: &SigningKey) -> String {
+    // hex encode public key for printing
+    let verifying_key_bytes = signing_key.verification_key().to_bytes();
+    hex::encode(verifying_key_bytes)
+}
+
+/// Get the private key from the signing key
+fn get_private_key_pretty(signing_key: &SigningKey) -> String {
+    let secret_key_bytes = signing_key.to_bytes();
+    hex::encode(secret_key_bytes)
+}
+
+/// Get the address from the signing key
+fn get_address_pretty(signing_key: &SigningKey) -> String {
+    // sha256 hash public key and take first 20 bytes to get address for printing
+    let verifying_key_bytes = signing_key.verification_key().to_bytes();
+    let mut hasher = Sha256::new();
+    hasher.update(verifying_key_bytes);
+    let address_bytes = hasher.finalize();
+    hex::encode(&address_bytes[..20])
```

Comment:
you can also do `proto::native::sequencer::v1alpha1::Address::from_verification_key(signing_key.verification_key()`